lib/dao.js

```diff
@@ -2334,6 +2335,12 @@ DataAccessObject.prototype.updateAttributes = function updateAttributes(data, op
     if (err) return cb(err);
     data = ctx.data;
 
+    if (strict) {
+      var keys = Object.keys(Model.definition.properties);
```

Comment:
It's documented at http://docs.strongloop.com/display/LB/Model+definition+JSON+file#ModeldefinitionJSONfile-Top-levelproperties. 

- `throw` Throw an error if unknown properties are present
- `validate` Report as an validation error (I prefer this)

See https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/model.js#L227-L230